invenio_app_ils/permissions.py

```diff
@@ -107,7 +107,11 @@ def loan_extend_circulation_permission(loan):
     if current_user.is_anonymous:
         abort(401)
 
-    if current_user.id == int(loan["patron_pid"]):
+    is_admin_or_librarian = (
+        has_request_context() and backoffice_permission().allows(g.identity)
```

Comment:
has_request_context() will be always `True` here

Reply:
Right, I will fix it.